autoortho/aoconfig.py

```diff
@@ -45,6 +45,43 @@ def __eq__(self, other):
         return NotImplemented
 
 
+def clean_cache(cache_dir, size_gb):
+
+    log.info(f"Cleaning up cache_dir {cache_dir}")
+
+    target_gb = max(size_gb, 10)
+    target_bytes = pow(2,30) * target_gb
+
+    cfiles = sorted(pathlib.Path(cache_dir).glob('**/*'), key=os.path.getmtime)
```

Comment:
We could use atime to prioritize areas where you recently where (aka images recently accessed).
```suggestion
    cfiles = sorted(pathlib.Path(cache_dir).glob('**/*'), key=os.path.getatime)
```
From my understanding `os.path.getatime()` should return mtime for file systems that don't support it so that seems safe.

Reply:
I agree that atime is really what we want here.

Reply:
I assumed it from what I am used with Linux `stat` on my noatime file systems...
You are right - we should check the other platforms. I've dug a bit into the python source an and I think the syscalls are [here](https://github.com/python/cpython/blob/main/Python/fileutils.c#L1233-L1284):  so `fstat()` on Unix / `GetFileInformationByHandle` on Windows. But I can not find an assertion that atime is always >= mtime, it's  totally up to the file system. So my assumption was wrong. Maybe this? 
```suggestion
    cfiles = sorted(pathlib.Path(cache_dir).glob('**/*'), key = lambda filePath: max(os.path.getatime(filePath), os.path.getmtime(filePath)))
```
(Python syntax is not my home turf so I might be missing a nicer way)

Reply:
Going to stick with mtime.  Atime may be updated whenever a filemanager browses the dir and other times.  Instead going to 'touch' the file to update mtime when a cache read occurs.